src/Control/Effect/Cull.hs

```diff
@@ -14,12 +14,11 @@ module Control.Effect.Cull
 , run
 ) where
 
-import Control.Applicative (Alternative(..))
 import Control.Effect.Carrier
 import Control.Effect.NonDet
 import Control.Effect.Reader
 import Control.Monad (MonadPlus(..))
-import Control.Monad.Fail
+import qualified Control.Monad.Fail as Fail
```

Comment:
`base-4.13` exports `MonadFail` from `Prelude`. We still support older versions of `base`, and I didn’t want to silence redundant import warnings, so I instead qualified the import so that it is, in fact, used.